File: Proyecto/vagrant/kotlin-concepts-service-api/kotlin-concepts-service-api/src/main/kotlin/com/puj/admincenter/domain/concepts/Concept.kt
package com.puj.admincenter.domain.concepts

import org.hibernate.annotations.GenericGenerator
import org.springframework.security.core.GrantedAuthority
import org.springframework.security.core.userdetails.UserDetails
import com.fasterxml.jackson.annotation.JsonIgnore
import java.io.Serializable
import java.util.*
import javax.persistence.*

@Entity
@Table(name = "concepts")
data class Concept(
    
    @Column(name = "id", nullable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int = 0,

    @Column(nullable = false, unique = true)
    val pxordx: String = "",

    @Column(nullable = false)
    val oldpxordx: String = "",

    @Column(nullable = false)
    val codetype: String = "",

    @Column(nullable = false)
    val concept_class_id: String = "",

    @Id
    @Column(nullable = false)
    val concept_id: Int = 0,

    @Column(nullable = false)
    val vocabulary_id: String = "",

    @Column(nullable = false)
    val domain_id: String = "",

    @Column(nullable = false)
    val track: String = "",

    @Column(nullable = true)
    val standard_concept: String = "",

    @Column(nullable = false)
    val code: String = "",

    @Column(nullable = false)
    val codewithperiods: String = "",

    @Column(nullable = false)
    val codescheme: String = "",

    @Column(nullable = false)
    val long_desc: String = "",

    @Column(nullable = true)
    val short_desc: String = "",

    @Column(nullable = true)
    val code_status: String = "",

    @Column(nullable = true)
    val code_change: String = "",

    @Column(nullable = true)
    val code_change_year: Int = 0,

    @Column(nullable = true)
    val code_planned_type: String = "",

    @Column(nullable = true)
    val code_billing_status: String = "",

    @Column(nullable = true)
    val code_cms_claim_status: String = "", 

    @Column(nullable = true)
    var sex_cd: String? = "", 

    @Column(nullable = true)
    val anat_or_cond: String = "", 

    @Column(nullable = true)
    val poa_code_status: String = "", 

    @Column(nullable = true)
    val poa_code_change: String = "", 

    @Column(nullable = true)
    val poa_code_change_year: String = "",

    @Column(nullable = true)
    val valid_start_date: String = "",

    @Column(nullable = true)
    val valid_end_date: String = "",

    @Column(nullable = true)
    val invalid_reason: String = "",

    @Column(nullable = true)
    val create_dt: Int = 0,

    @Column(nullable = true)
    val concept_status: Boolean = true
)

File: Proyecto/WebDev/WebDev.Application/Models/Concept.cs
using System.ComponentModel.DataAnnotations;

namespace WebDev.Application.Models
{
    public class Concept
    {
        [Key]
        public int id { get; set; }
        public string pxordx { get; set; }
        public string oldpxordx { get; set; }
        public string codeType { get; set; }
        public string concept_Class_Id { get; set; }
        public int concept_Id { get; set; }
        public string vocabulary_Id { get; set; }
        public string domain_Id { get; set; }
        public string track { get; set; }
        public string standard_Concept { get; set; }
        public string code { get; set; }
        public string codeWithPeriods { get; set; }
        public string codeScheme { get; set; }
        public string long_Desc { get; set; }
        public string short_Desc { get; set; }
        public string code_Status { get; set; }
        public string code_Change { get; set; }
        public int code_Change_Year { get; set; }
        public string code_Planned_Type { get; set; }
        public string code_Billing_Status { get; set; }
        public string code_Cms_Claim_Status { get; set; }
        public string sex_Cd { get; set; }
        public string anat_Or_Cond { get; set; }
        public string poa_Code_Status { get; set; }
        public string poa_Code_Change { get; set; }
        public string poa_Code_Change_Year { get; set; }
        public string valid_Start_Date { get; set; }
        public string valid_End_Date { get; set; }
        public string invalid_Reason { get; set; }
        public int create_Dt { get; set; }
    }
}
